src/UserMenu.java

```diff
@@ -70,18 +68,36 @@ public boolean createSavingsAccount() {
 		return true;
 	}
 	
-	/* TODO Create checking account and display account menu */
-	public boolean createCheckingAccount() {
-		System.out.println("TO BE IMPLEMENTED: Create Checking Account");
-		return true;
+	/**
+	 * Creates a new checking account for the current user and displays the new account's menu
+	 */
+	public void createCheckingAccount() {
+		Account checkingAccount = new Account(0);
+		User.currentUser.addAccount(checkingAccount);
+		AccountMenu accountMenu = new AccountMenu(checkingAccount);
+		accountMenu.runMenu();
 	}
 	
-	/* TODO: Display list of accounts for user to select from and 
-	 * allow user to pick one to display the account menu
+	/**
+	 * Displays the accounts of the current user and asks them which account they want to select
 	 */
-	public boolean selectAccount() {
-		System.out.println("TO BE IMPLEMENTED: Select Account");
-		return true;
+	public void selectAccount() {
```

Comment:
Can you separate this into smaller functions?

Reply:
Sure

Reply:
Done